js/util.js

```diff
@@ -14,8 +14,22 @@ window.util = (function () {
         action();
       }
     },
-    getRandom: function (array) {
-      return array[Math.floor(Math.random() * array.length)];
+    getRandomValue: function (min, array) {
+      var max = array;
+      return Array.isArray(array) === true ?
```

Comment:
```suggestion
      return Array.isArray(array) ?
```

лучше параметр функции `array` заменить на `arr`, хоть js регистрозависимый, `array `воспринимается как тип